lib/Mail.php

```diff
@@ -623,7 +623,12 @@ public function sendWithoutRendering(\Swift_Mailer $mailer = null)
 
         if ($event->hasArgument('mailer')) {
             $mailer = $event->getArgument('mailer');
-            $mailer->send($this);
+            try {
+                $mailer->send($this, $emailaddress);            
+            } catch (\Exception $e){
+                $mailer->getTransport()->stop();
+                throw new \Exception($emailaddress[0].' - '.$e->getMessage());
+            }
```

Comment:
This should work well, without sleep(), according to swiftmailer docs:

/**
     * Check if this Transport mechanism is alive.
     *
     * If a Transport mechanism session is no longer functional, the method
     * returns FALSE. It is the responsibility of the developer to handle this
     * case and restart the Transport mechanism manually.
     *
     * @example
     *
     *   if (!$transport->ping()) {
     *      $transport->stop();
     *      $transport->start();
     *   }
     *
     * The Transport mechanism will be started, if it is not already.
     *
     * It is undefined if the Transport mechanism attempts to restart as long as
     * the return value reflects whether the mechanism is now functional.
     *
     * @return bool TRUE if the transport is alive
     */